app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ReviewsController < ApplicationController
+  def new
+    @spot = Spot.find params[:spot_id]
+    @review = Review.new
+    @ratings = [1, 2, 3, 4, 5]
+  end
+
+  def create
+    session[:user_id] ||= "1"
```

Comment:
You shouldn't be setting a new user in the session, if one doesn't already exist. Do you want to require a user to be signed in for this action? If so, check out devise's `authenticate_user!` method. You can access the current user by using the pre-defined method `current_user`. Use that to assign a user to the `@review` object!